server.go

```diff
@@ -0,0 +1,133 @@
+package veneur
+
+import (
+	"net"
+	"sync"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/Sirupsen/logrus"
+	"github.com/getsentry/raven-go"
+)
+
+type Server struct {
+	Workers []*Worker
+
+	statsd *statsd.Client
+	logger *logrus.Logger
+	sentry *raven.Client
+
+	Hostname string
+	Tags     []string
+
+	DDHostname string
+	DDAPIKey   string
+
+	UDPAddr     *net.UDPAddr
+	RcvbufBytes int
+}
+
+func NewFromConfig(conf Config) (ret Server, err error) {
+	ret.Hostname = conf.Hostname
+	ret.Tags = conf.Tags
+	ret.DDHostname = conf.APIHostname
+	ret.DDAPIKey = conf.Key
+
```

Comment:
This isn't relevant to your change directly, but since were here… I'd love if this method emitted the whole config object (maybe without the DD API key) in to the log at startup to confirm settings and stuff. I like seeing how many workers and other options as a sanity check. Feel free to disregard!